Floofbot/Modules/Utilities.cs

```diff
@@ -67,14 +71,26 @@ public async Task Avatar([Remainder] IGuildUser user = null)
                 user = (IGuildUser)Context.User;
 
             var avatarUrl = user.GetAvatarUrl(ImageFormat.Auto, 512);
-            EmbedBuilder builder = new EmbedBuilder(){
+            EmbedBuilder builder = new EmbedBuilder()
+            {
                 Description = $"🖼️ **Avatar for:** {user.Mention}\n",
                 ImageUrl = avatarUrl,
-                Color = Color.Magenta
+                Color = EMBED_COLOR
 
             };
             await Context.Channel.SendMessageAsync("", false, builder.Build());
+        }
 
+        [Command("say")]
+        [Summary("Reapeats a message")]
+        public async Task RepeatMessage([Remainder] string message =null)
+        {
+            EmbedBuilder builder = new EmbedBuilder()
+            {
+                Description = message,
```

Comment:
I would handle null messages. Also wary of @ tags in embeds.

Reply:
The message is embed.  @everyone and other pings do not go through